source/overview/faq.rst

```diff
@@ -366,12 +366,12 @@ The following chart highlights the end user features of Mattermost and their sup
     "Markdown", "X", "X", "O"
     "Emojis", "X", "X", "X"
     "Emoji reactions", "X", "X", "X"
-    "Viewing emoji reactions", "X", "X", "v1.13"
+    "Viewing emoji reactions", "X", "X", "X"
     "File sharing", "X", "X", "X"
     "@ mentions", "X", "X", "X"
-    "Hashtags", "X", "X", "v1.13"
+    "Hashtags", "X", "X", "X"
     "Search (with in:, from:, before:, on: and after:)", "X", "X", "X"
-    "Search highlighting", "X", "X", "v1.15"
+    "Search highlighting", "X", "X", "???"
```

Comment:
We're shipping mention and reply highlighting - does this cover search highlighting as well?

Reply:
No not yet, that'll be 1.17 hopefully